### PR TITLE
Test for kwargs only

### PR DESCRIPTION
**Pull request by aaltat**

### CODE REVIEW

File: atest/tests_types.robot [hunk 1]
```diff
@@ -53,6 +53,17 @@ Keyword Annonations And Robot Types Defined
     ${return} =    DynamicTypesAnnotationsLibrary.Keyword Robot Types And Bool Defaults    tidii    111
     Should Match Regexp    ${return}    tidii: <(class|type) 'str'>, 111: <(class|type) 'str'>
 
+Keyword Arguments Only As String
+    [Tags]    py3
```
Comment (pekkaklarck) [inline atest/tests_types.robot]: `**kwargs` works just fine with Python 2.

File: atest/tests_types.robot [hunk 2]
```diff
@@ -53,6 +53,17 @@ Keyword Annonations And Robot Types Defined
...
+    ${return} =    DynamicTypesAnnotationsLibrary.Keyword Kwargs Only    k1=1    k2=2
+    Should Be Equal    ${return}    k1: 1, k2: 2
+
+Keyword Arguments Only As Dictionary
+    [Tags]    py3
+    ${args}        Create Dictionary   k1=1    k2=2
+    ${return} =    DynamicTypesAnnotationsLibrary.Keyword Kwargs Only    &{args}
+    Should Be Equal    ${return}    k1: 1, k2: 2
```
Comment (pekkaklarck) [inline atest/tests_types.robot]: I'm not sure is testing with `name=value` and `&{dict}` necessary. It feels like testing Robot Framework functionality more than PythonLibCore.

File: utest/test_robotlibcore.py [hunk 3]
```diff
@@ -161,3 +165,19 @@ class OldStyle:
         HybridCore([OldStyle()])
     assert str(exc_info.value) == \
            "Libraries must be modules or new-style class instances, got old-style class 'OldStyle' instead."
+
+
+@pytest.mark.skipif(sys.version_info[0] == 2, reason='Only applicable on Py 3')
+@pytest.mark.skipif(robot__version >= '3.2', reason='For RF 3.1')
```
Comment (pekkaklarck) [inline utest/test_robotlibcore.py]: As already commented earlier, `**kwargs` works with Python 2 just fine. It also works also with earlier Robot Framework versions. I'm also slightly surprised if there weren't tests for it already.

Comment (aaltat) [reply utest/test_robotlibcore.py]: There was a test that had `*varargs` and `**kwargs` but not test for kwargs alone.

**Comment by aaltat** [comment]
I think we do not need this complex stuff.

**Pull request closed without merging**